woof-distro/x86_64/debian/bullseye64/DISTRO_PKGS_SPECS-debian-bullseye

```diff
@@ -172,6 +175,7 @@ yes|glibc_l10n|libc-l10n|exe,dev,doc,nls||deps:yes
 no|gmeasures||exe,dev>null,doc,nls
 yes|gmp|libgmp10,libgmpxx4ldbl,libgmp-dev|exe,dev,doc,nls||deps:yes #in precise, this was only in devx, but abiword needs it.
 no|gnome-doc-utils|gnome-doc-utils|exe>dev,dev,doc,nls|+python-libxml2
+yes|gnome-icon-theme|gnome-icon-theme|exe>null,dev>null,doc>null,nls>null
```

Comment:
Is this really needed? What icons are missing? I can fix that.

Reply:
No icons are missing, but I wanted to prevent installation of this huge package. It's a dependency of Blueman. 

PMaterial is not 100% complete (i.e some missing pavucontrol iucons), but close enough and much smaller, so there's no reason to install this package.

Reply:
Ok, I'll try to add icons to all 3 themes over the week end so we don't need packages like adwaita icons or gnome icons.

Reply:
I do know that some microphone icons are missing - gtk3iconbrowser is my friend there :smile:

Reply:
Sorry, totally missed the `>null` redirect - oops